packages/react-native/src/evm/constants/walletConnect.ts

```diff
@@ -0,0 +1,2 @@
+export const TW_WC_PROJECT_ID = '145769e410f16970a79ff77b2d89a1e0'
```

Comment:
oooh so we can let ppl use this one by default? is there any limitations?

Reply:
I'm pretty sure they will throttle by projectID in the future